sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -358,6 +358,14 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ADAPTIVE_EXECUTION_FORCE_APPLY = buildConf("spark.sql.adaptive.forceApply")
```

Comment:
`spark.sql.adaptive.forceApply.enabled`?
cc @gatorsmile

Reply:
No... that sounds weird.

Reply:
@maryannxue . `.enabled` is a general guideline for boolean flag from @gatorsmile .
> No... that sounds weird.

Reply:
it's usually `xxx.featureName.enabled`, but `forceApply` is a verb. For example, `spark.sql.join.preferSortMergeJoin`

Reply:
Oh. Got it. I misunderstood the rule at that part until now. My bad. Thank you, @cloud-fan and @maryannxue .

Reply:
Then why don't we make it a feature name and add `.enabled` to be consistent?

Reply:
Such configurations `spark.sql.join.preferSortMergeJoin` were added long ago but this is new where it's free to change.

Reply:
do we have a policy that config name must be `xxx.featureName.enabled`? At least for internal configs we follow PR author's personal preference AFAIK.

Reply:
We should then stop renaming the configurations to add `.enabled` (e.g., https://github.com/apache/spark/pull/27346, https://github.com/apache/spark/pull/27210, https://github.com/apache/spark/pull/26694). `reuse`, `ignore`, and `fail`  can be a verb either.

Reply:
@gatorsmile do we need to add ".enabled" post-fix to all boolean configs?

Reply:
After this PR, it would be great if we have a documented policy for this, @gatorsmile and @cloud-fan .

Reply:
Thank you for the references, @HyukjinKwon .